requirements.txt

```diff
@@ -1,5 +1,5 @@
-pyrovision >= 0.1.2
-python-dotenv >= 0.15.0
-requests>=2.25.1
+pyrovision
```

Comment:
At least a lower bound would be nice don't you think?

Reply:
We have an issue with our version number, we get `pyrovision--version-.-.-0.1.2a0-288795a` therefore the condition `pyrovision>xx` will always fail

Reply:
I think this will fix the problem: https://github.com/pyronear/pyro-vision/pull/133

Reply:
Thanks ! But I was thinking, maybe it's better to install provision from the repo. We need all the latest updates

Reply:
Those are parallel topics (means of installation vs. version specifier), but I'd say:
- we need a version specifier for production repo :sweat_smile: 
- regarding the means of installation, installing from the repo means that if a bug slips in a commit, it will have it. if we put the normal one, we can always install from the repo before running that installation and have the choice. We'll need to make a release of pyrovision soon anyway 

What do you think?

Reply:
Yes, I see what you mean. I think we need to make a 0.1.3 release and define this version as a minimum to include recent bug fixes.

Reply:
can you review the [open_fire_v2 PR](https://github.com/pyronear/pyro-vision/pull/121) to include it in the release ?